src/TypeScript.fs

```diff
@@ -4349,14 +4349,14 @@ module Ts =
         abstract wildcardDirectories: MapLike<WatchDirectoryFlags> with get, set
 
     type [<AllowNullLiteral>] ModuleResolutionHost =
-        abstract fileExists: fileName: string -> bool
-        abstract readFile: fileName: string -> string option
+        abstract fileExists: (string -> bool) with get,set
+        abstract readFile: (string -> string option) with get,set
         abstract trace: s: string -> unit
-        abstract directoryExists: directoryName: string -> bool
+        abstract directoryExists: (string -> bool) with get,set
         /// Resolve a symbolic link.
         abstract realpath: path: string -> string
         abstract getCurrentDirectory: unit -> string
-        abstract getDirectories: path: string -> ResizeArray<string>
+        abstract getDirectories: (string -> ResizeArray<string>) with get,set
 
```

Comment:
Is there any better way to make function property settable?

Reply:
Hum, I don't think. 
For me it's fine like that.

Reply:
Not sure... if this is working it's probably the best solution :)